app/Services/Manager.php

```diff
@@ -92,6 +92,77 @@ public function exportCSV($model, $hasReportback = false)
     }
 
     /**
+     * Build leaderboard data for a given Competition
+     *
+     * @param Competition $competition
+     * @param User $users
+     * @return array $leaderboard
+     */
+    public function getLeaderboard($competition, $users)
+    {
+        $rows = [];
+
+        // Get all users in bulk
+        $parameters['ids'] = implode(',', $users->pluck('id')->all());
```

Comment:
If I pass an array of ID's into the getAllUsers how does it know its an array of ID's? in the code we're referencing the $parameters var was just an array of ID's like it is here https://github.com/DoSomething/gladiator/blob/77ca2e1102d8af1a2878fbe79c1d56eb9f054c01/app/Services/Manager.php#L199